packages/truffle-core/lib/mnemonics/mnemonic.js

```diff
@@ -1,44 +1,44 @@
 /**
-* @module mnemonic;
-* @requires module:truffle-config
-* @requires module:seedrandom
-* @requires module:bip39
-* @requires module:ethereumjs-wallet/hdkey
-* @requires module:crypto
-*/
+ * @module mnemonic;
+ * @requires module:truffle-config
+ * @requires module:seedrandom
+ * @requires module:bip39
+ * @requires module:ethereumjs-wallet/hdkey
+ * @requires module:crypto
+ */
 
-const Config = require('truffle-config');
+const Config = require("truffle-config");
 const defaultUserConfig = Config.getUserConfig();
-const bip39 = require('bip39'); 
-const hdkey = require('ethereumjs-wallet/hdkey');
-const crypto = require('crypto');
+const bip39 = require("bip39");
+const hdkey = require("ethereumjs-wallet/hdkey");
+const crypto = require("crypto");
 
 const mnemonic = {
-
   /**
-  * gets user-level mnemonic from user config, and if missing generates a new mnemonic
-  * @returns {String} mnemonic
-  */
-  getOrGenerateMnemonic: function () {
+   * gets user-level mnemonic from user config, and if missing generates a new mnemonic
+   * @returns {String} mnemonic
+   */
+  getOrGenerateMnemonic: function() {
     let mnemonic;
     const userMnemonicExists = defaultUserConfig.get("mnemonic");
```

Comment:
I know this is already existing code but I object to the name for this.  If we use a variable name like `userMnemonicExists` I think that it should be a boolean.  I would propose that this variable gets set to something like `userMnemonic` or just `mnemonic`.  So it could look like `if (mnemonic) { ...`
And actually that would simplify the logic a bit as you wouldn't need the else.

Reply:
I'm about to update the `pluggable-truffle` branch to clean up the extra code in the plugin PRs, so cc @fainashalts

Reply:
Totally fine with me to change this, good call @eggplantzzz